fastlane/Fastfile

```diff
@@ -61,6 +61,56 @@ lane :github_release do |options|
   )
 end
 
+desc "Update dependencies to latest GitHub releases and opens a PR"
+lane :open_pr_upgrading_dependencies do |options|
+  dry_run = options[:dry_run]
+  
+  new_versions = update_native_dependencies_to_latest
+  
+  repo_status = sh("git status --porcelain")
+
+  repo_clean = repo_status.empty?
+
+  if repo_clean
+    UI.success('Git status is clean, nothing to update! 💪')
+    next
+  end
+
+  new_ios_version = new_versions[:new_ios_version]
+  new_android_version = new_versions[:new_android_version]
+
+  if dry_run
+    UI.message("ℹ️  Nothing more to do since dry_run: true.")
+    UI.message("ℹ️  New iOS dependency version is #{new_ios_version}")
+    UI.message("ℹ️  New Android dependency version is #{new_android_version}")
+  else
+    branch = "update-ios-#{new_ios_version}-android-#{new_android_version}"
```

Comment:
Note that it's possible this only updates one of the SDKs. I don't think it's a big deal though